std/php7/Global.hx

```diff
@@ -82,4 +83,133 @@ extern class Global {
 	**/
 	@:overload(function(original:String,alias:String):Bool {})
 	static function class_alias( original:String, alias:String, autoload:Bool ) : Bool ;
+
+	/**
+		@see http://php.net/manual/en/function.count.php
```

Comment:
Nope. It's std lib code style. E.g. https://github.com/HaxeFoundation/haxe/blob/development/std/Array.hx#L22